src/go/rpk/pkg/cli/cmd/cluster/config/set.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v3"
+)
+
+func newSetCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <key> <value>",
```

Comment:
Would it be better to have a StringSlice flag that allows setting multiple key/value pairs at a time? `rpk topic create` has `-c` to set config key=value pairs, `rpk topic alter-config` has `--set key=value` to set many configs at a time (and `--delete`).

Reply:
While it's not super complicated to do something like that in terms of the code, I'm not sure that could==should in this case.  I like the simplicity of saying:
- `edit` is what you use for interactive changes
- `import` does a whole configuration in one go, it's what you use if you already know everything about your config (e.g. paving out a new cluster the same as another cluster)
- `set` does one value at a time, it's provided for use in shell scripts.

I really want to funnel interactive use cases toward the `edit` path, because they get inline docs when they use that -- keeping `set` quite spartan serves that goal.